notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
<p>the links are not correct, should be the link when this notebook is merged to main</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@morgandu done.